app/src/main/java/org/wikipedia/page/Page.java

```diff
@@ -29,6 +33,9 @@ public Page(@NonNull PageTitle title, @NonNull List<Section> sections,
         return sections;
     }
 
+    @NonNull public void setSections(List<Section> sections) {
```

Comment:
A `void` function should not have annotation, and please add `@NonNull` to the section parameter.